components/src/components.d.ts

```diff
@@ -60,6 +74,8 @@ declare global {
   interface HTMLElementTagNameMap {
     'pulumi-choosable': HTMLPulumiChoosableElement;
     'pulumi-chooser': HTMLPulumiChooserElement;
+    'pulumi-example': HTMLPulumiExampleElement;
```

Comment:
Just so I understand, the prefix `pulumi-` here is just for the component and not for the actual shortcode, correct?

Reply:
Yep, that's right.